src/main/resources/data-repository-openapi.yaml

```diff
@@ -1898,6 +2051,138 @@ definitions:
         items:
           $ref: '#/definitions/FileModel'
 
+  LoadTagModel:
+    type: string
+    description: |
+      client-specified tag for a data or file load. If no id is specified, we use the
+      string form of the job create time as the tag.
+
+  BulkLoadRequestModel:
+    description: |-
+      Body of a bulk file load request. We provide two ways of specifying the files to
+      load. One is to include an array of BulkLoadFileModel objects in this
+      object. The other is to provide a file containing the JSON form of a
+      BulkLoadFileModel. While we expect users to pick one or the other, the code will
+      process both inputs and load all of the files. If neither is specified, it runs
+      really, really fast!
+      The results of a bulk load are always stored into the dataset tabular data store
+      and can be retrieved directly from there, or via the GET .../bulk/{loadTag}
+      They can be cleaned up with the DELETE .../bulk/{loadTag}
+    type: object
+    required:
+      - profileId
+      - loadTag
+    properties:
+      profileId:
+        $ref: '#/definitions/UniqueIdProperty'
+        description: The profile id to use for these files
+      loadTag:
+        $ref: '#/definitions/LoadTagModel'
+      maxFailedFileLoads:
+        type: integer
+        default: 0
+        description: max number of failed file loads before stopping
+      returnArrayResults:
+        type: boolean
+        default: false
+        description: |-
+          If true, return the load results in an array in the result model.
```

Comment:
I'm unsure about the usefulness of this toggle, especially if it's dangerous. If returning all the results in an array is a runtime bomb and the other way to look up results is safe bc pagination, then I think it's fine to have only summary stats in the top-level result. (Disclaimer: it's the same API pattern we used for our X->GCP transfer system, so I'm definitely biased)